baseplate/clients/memcache/__init__.py

```diff
@@ -129,9 +130,35 @@ class MemcacheContextFactory(ContextFactory):
 
     """
 
-    def __init__(self, pooled_client: PooledClient):
+    PROM_PREFIX = "bp_memcached_pool"
+    PROM_LABELS = ["pool"]
+
+    promTotalConnections = Gauge(
+        f"{PROM_PREFIX}_size",
+        "Maximum size of this pool",
+        PROM_LABELS,
+    )
+
+    promUsedConnections = Gauge(
+        f"{PROM_PREFIX}_in_use",
+        "Number of connections in this pool currently in use",
+        PROM_LABELS,
+    )
+
+    promFreeConnections = Gauge(
+        f"{PROM_PREFIX}_free",
+        "Number of free connections in this pool",
+        PROM_LABELS,
+    )
+
+    def __init__(self, pooled_client: PooledClient, name: str = "memcache"):
```

Comment:
:no_bell: Would `None` or `""` work as a default since the metric name already has "memcache" in it?